README.md

```diff
@@ -1229,9 +1230,423 @@ python evaluate.py --num-conversations 5
+
+```bash
+# Set your namespace
+export NAMESPACE=your-namespace
+
+# Create Jaeger deployment
+cat <<EOF | oc apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jaeger
+  namespace: ${NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jaeger
+  template:
+    metadata:
+      labels:
+        app: jaeger
+    spec:
+      containers:
+      - name: jaeger
+        image: jaegertracing/all-in-one:latest
+        env:
+        - name: COLLECTOR_OTLP_ENABLED
+          value: "true"
+        ports:
+        - containerPort: 16686
+          name: ui
+          protocol: TCP
+        - containerPort: 4318
+          name: otlp-http
+          protocol: TCP
+        - containerPort: 4317
+          name: otlp-grpc
+          protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: jaeger-collector
+  namespace: ${NAMESPACE}
+spec:
+  ports:
+  - name: otlp-http
+    port: 4318
+    protocol: TCP
+    targetPort: 4318
+  - name: otlp-grpc
+    port: 4317
+    protocol: TCP
+    targetPort: 4317
+  selector:
+    app: jaeger
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: jaeger-query
+  namespace: ${NAMESPACE}
+spec:
+  ports:
+  - name: ui
+    port: 16686
+    protocol: TCP
+    targetPort: 16686
+  selector:
+    app: jaeger
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: jaeger-ui
+  namespace: ${NAMESPACE}
+spec:
+  to:
+    kind: Service
+    name: jaeger-query
+  port:
+    targetPort: ui
+  tls:
+    termination: edge
+    insecureEdgeTerminationPolicy: Redirect
+EOF
+```
+
+**Step 2: Verify Deployment**
+
+```bash
+# Check pod status
+oc get pods -n $NAMESPACE -l app=jaeger
+
+# Get Jaeger UI URL
+export JAEGER_UI_URL=$(oc get route jaeger-ui -n $NAMESPACE -o jsonpath='{.spec.host}')
+echo "Jaeger UI: https://$JAEGER_UI_URL"
+```
+
+**Step 3: Configure Self-Service Agent**
+
+Set the OTLP endpoint to point to your Jaeger collector:
+
+```bash
+export OTEL_EXPORTER_OTLP_ENDPOINT="http://jaeger-collector.${NAMESPACE}.svc.cluster.local:4318"
+
+# Deploy or upgrade with tracing enabled
+make helm-upgrade NAMESPACE=$NAMESPACE \
+  EXTRA_HELM_ARGS="--set-string otel.endpoint=$OTEL_EXPORTER_OTLP_ENDPOINT"
+```
+
+**Limitations:**
+- In-memory storage only (traces lost on restart)
+- Not suitable for production workloads
+- No data persistence or high availability
+- Limited to single namespace deployment
+
+**Documentation:**
+- [Jaeger Getting Started](https://www.jaegertracing.io/docs/latest/getting-started/)
+- [Jaeger Deployment Guide](https://www.jaegertracing.io/docs/latest/deployment/)
+
+---
+
+#### Option 2: OpenShift Observability with Tempo (Production)
+
+For production deployments, use the Red Hat OpenShift distributed tracing platform based on Tempo. This provides enterprise-grade features including persistent storage, high availability, multi-tenancy, and integration with the OpenShift observability stack.
```

Comment:
I think much of this comes from the observability quickstart? We should probably reference it right here with something like "if you want a more detailed information/understanding you can check out this quickstart, we've extracted the key steps below"